math/src/field/element.rs

```diff
@@ -103,6 +104,16 @@ where
     }
 }
 
+/// Sum operator for field elements
+impl<F> Sum<FieldElement<F>> for FieldElement<F>
+where
+    F: IsField,
+{
+    fn sum<I: Iterator<Item = Self>>(iter: I) -> Self {
+        iter.fold(Self::zero(), |a, b| a + b)
```

Comment:
Could we change the parameters name to `addend` and `augend` like this?
```suggestion
        iter.fold(Self::zero(), |augend, addend| augend + addend)
```